Microsoft.SCIM.WebHostSample/Provider/InMemoryUserProvider.cs

```diff
@@ -186,32 +165,34 @@ public override Task<Resource[]> QueryAsync(IQueryParameters parameters, string
                                     string.Format(SystemForCrossDomainIdentityManagementServiceResources.ExceptionFilterOperatorNotSupportedTemplate, andFilter.FilterOperator));
                             }
 
-                            users =
-                                users.Where(
-                                    item =>
-                                       item.Active == bool.Parse(andFilter.ComparisonValue)).ToList();
+                            bool active = bool.Parse(andFilter.ComparisonValue);
+                            predicateAnd = predicateAnd.And(p => p.Active == active);
+
                         }
 
-                        // LastModified filter
+                        //
```

Comment:
@CecilyK could you update this comment, the value should be: //LastModified filter

Reply:
Done :)